blog/2024-04-30-nushell_0_93_0.md

```diff
@@ -191,21 +204,92 @@ This frees up stdio for interaction with the user's terminal, which makes plugin
 
 We also added [engine calls](/contributor-book/plugin_protocol_reference.md#enterforeground-engine-call) for moving the plugin into the terminal controlling (foreground) process group, mainly for Unix platforms. This was also necessary to fully support terminal UI plugins, after the last release's new persistence functionality caused plugins to run in a new, background process group by default.
 
+## Improved parsing of input/output types [[toc](#table-of-content)]
+
+Thanks to [@texastoland](https://github.com/texastoland) in [#12339](https://github.com/nushell/nushell/pull/12339),
+the input/output types for custom commands can now extend across multiple lines.
+
+```nushell
+def demo [--verbose]: [
+  nothing -> nothing
+  nothing -> string
```

Comment:
As long as we have no way to resolve those diverging signatures during type check we should probably reject them with an error that you have to use a more general type.